pkg/ipam/service/allocator/utils.go

```diff
@@ -11,7 +11,7 @@ import (
 
 // countBits returns the number of set bits in n
 func countBits(n *big.Int) int {
-	var count int = 0
+	var count = 0
```

Comment:
nit: this is the zero value for that type. `var count int` should suffice...

Reply:
thanks